LDAP/ldap2s3/README.md

```diff
@@ -0,0 +1,39 @@
+# LDAP2S3
+
+This script queries and convert LDAP data from Mozilla LDAP's specific schema to JSON files that get stored in S3
+buckets.
+
+# Security
+
+Group data is used for access control and thus important. For that reason, and while there's additional checks made
+outside of this script, the script ensures:
+
+- JSON output is signed
```

Comment:
that's not actually true. i wonder if we can make it true in a way that's operationally functional